dashboard/db/migrate/20170612195103_change_plc_course_units_unit_description.rb

```diff
@@ -0,0 +1,14 @@
+class ChangePlcCourseUnitsUnitDescription < ActiveRecord::Migration[5.0]
+  def change
+    reversible do |dir|
+      change_table :plc_course_units do |t|
+        dir.up do
+          t.change :unit_description, :text
+        end
+        dir.down do
+          t.change :unit_description, :string
+        end
+      end
+    end
+  end
```

Comment:
The source of truth for the data in this table should be `config/locales/scripts.en.yml`. If there's some issue with a course that was deleted from the config file but not the table, we can always recover it from version control.